src/frontend/components/VideoPlayer/index.tsx

```diff
@@ -160,6 +161,7 @@ const VideoPlayer = ({
       {transcripts.length > 0 && (
         <Transcripts transcripts={transcripts as TimedTextTranscript[]} />
       )}
+      {video.live_state !== null && video.xmpp && <Chat video={video} />}
```

Comment:
I would pass only `video.xmpp` as Chat component do not need the full video object. It helps to understand which data consume the component and also in general prevent some performance issue due to extra rendering.

Furthermore,  you should have to create a type do describe xmpp configuration object.